src/reducers/index.js

```diff
@@ -1,15 +1,15 @@
-import {uuidv1 as uuid} from 'uuid/v1'
+import {v1 as uuid} from 'uuid'
```

Comment:
This is a interesting library to use for Node.js.  I have not seen it or used it before. What are some of the downfalls for using it? I would need to research it more. Otherwise I would say you can use the string name + Math.random() to get a more random uuid for the todo id.